src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptor.java

```diff
@@ -285,11 +265,7 @@ final class SandboxInterceptor extends GroovyInterceptor {
             if (whitelist.permitsMethod(getterMethod, receiver, noArgs)) {
                 return super.onGetProperty(invoker, receiver, property);
             } else if (rejector == null) {
```

Comment:
This test for `null` isn't needed – initialized with `null` (line 260) and not modified. Is it safe to remove or should it be kept in case the code is restructured at a later point?

Reply:
Hmm. It does look safe to remove. I wonder why SpotBugs is not catching this though.